<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewVocab</class>
 <widget class="QMainWindow" name="NewVocab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widgetForCreateNewVocab" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>141</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>251</width>
       <height>135</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="layoutForCreateNewVocab">
      <item>
       <widget class="QLabel" name="labelForName">
        <property name="text">
         <string>Nom du nouveau dictionnaire</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditForName"/>
      </item>
      <item>
       <widget class="QLabel" name="labelNumberColumn">
        <property name="text">
         <string>Nombre de colonne</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditNumberOfColumn"/>
      </item>
      <item>
       <widget class="QPushButton" name="firstButtonForCreateVocab">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
